libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/FreeDriveMapBinder.kt

```diff
@@ -17,6 +18,10 @@ internal class FreeDriveMapBinder(
     override fun bind(mapView: MapView): MapboxNavigationObserver {
         return navigationListOf(
             LocationPuck(mapView),
+            RouteLineComponent(
```

Comment:
This component will be removed from `FreeDriveBinder` before merging this PR. It's only here to test camera in the interim

Reply:
Removed.